README.adoc

```diff
@@ -42,37 +45,55 @@ and configure a _Run/Debug configuration_ like this:
 * Working directory: `$MODULE_DIR$` (must resolve to optaplanner-examples directory)
 * Use classpath of module: `optaplanner-examples`
 
-=== Starter issues
+### Starter issues
 
 If you're just starting out with OptaPlanner and want to contribute,
 take a look at our https://issues.redhat.com/issues/?jql=project%20%3D%20PLANNER%20AND%20status%20in%20(Open%2C%20Reopened)%20AND%20labels%20%3D%20starter%20ORDER%20BY%20priority%20DESC[starter issues].
 They're specifically chosen to be easier for first time contributors.
 
-== Developing Drools, OptaPlanner and jBPM
+## Contributing to OptaPlanner
+
+This is an open source project, and you are more than welcomed to contribute!:heart:
+
+### Getting into
 
-*If you want to build or contribute to a kiegroup project, https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/main/README.md[read this document].*
+Working on a contribution or want to start an opinionated discussion? +
+Join our https://groups.google.com/g/optaplanner-dev[discussion] or mail directly through optaplanner-dev@googlegroups.com
```

Comment:
This may sound slightly better:
```suggestion
If you want to contribute or start an opinionated discussion, join our https://groups.google.com/g/optaplanner-dev[discussion] or send an e-mail directly to optaplanner-dev@googlegroups.com.
```